### PR TITLE
Add singularity resources

### PR DESCRIPTION
**Pull request by jaimesouza**
Update Makefile to download singularity .deb and .rpm packages
from Github Releases, and add singularity-deb and singularity-rpm
as slurmd resources.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,29 +11,33 @@ clean: ## Remove all deployed applications
 ${ETCD}: ## Download etcd resource
 	wget https://github.com/etcd-io/etcd/releases/download/v3.5.0/${ETCD}
 
+singularity: ## Donwload singularity .deb and .rpm
+	wget https://github.com/sylabs/singularity/releases/download/v3.10.2/singularity-ce_3.10.2-focal_amd64.deb
+	wget https://github.com/sylabs/singularity/releases/download/v3.10.2/singularity-ce-3.10.2-1.el7.x86_64.rpm
+
```
Comment (heitorPB) [inline Makefile:17]:
```suggestion
SINGULARITY_DEB=singularity-ce_3.10.2-focal_amd64.deb
SINGULARITY_RPM=singularity-ce-3.10.2-1.el7.x86_64.rpm

${SINGULARITY_DEB}:
	wget https://github.com/sylabs/singularity/releases/download/v3.10.2/${SINGULARITY_DEB}

${SINGULARITY_RPM}:
	wget https://github.com/sylabs/singularity/releases/download/v3.10.2/${SINGULARITY_RPM}

singularity: ${SINGULARITY_DEB} ${SINGULARITY_RPM} ## Donwload singularity .deb and .rpm

```

Comment (heitorPB) [reply Makefile:17]: This way we don't download the resources on all `make` commands.

**Comment by jaimesouza** [comment]
@heitorPB Is it approved now?

**Review by heitorPB — APPROVED** [review]
Nice work! Thank you.

**Pull request merged**